### PR TITLE
remove duplicate _get_cancel_requested

### PR DESCRIPTION
**Pull request by madhur-tandon**
- [ ] I have added the tests to cover my changes.
- [x] I have updated the documentation and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by madhur-tandon** [comment]
CC: @kessler-frost @santoshkumarradha @AlejandroEsquivel 
This should be ready as well.

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 
 - Duplicate mocks for `UI_SRVDIR`
+- Duplicate `_get_cancel_requested` method
```
Comment (kessler-frost) [inline CHANGELOG.md:30]: Can you also mention the file here, i.e `covalent_dispatcher/_core/runner.py`? Thanks!

**Review by kessler-frost — CHANGES_REQUESTED** [review]
Looks good 👍

File: CHANGELOG.md (hunk 1 above)
Comment (madhur-tandon) [reply CHANGELOG.md]: Thanks @kessler-frost, done!

**Comment by madhur-tandon** [comment]
@kessler-frost I have edited the changelog 😅

**Review by kessler-frost — APPROVED** [review]

**Pull request merged**